in_progress/mission-statement.md

```diff
@@ -0,0 +1,51 @@
+# Mission Statement
+
+## Values
+
+* Efficiency – Work the largest amount of data possible given available resources. We are not just targeting the largest datasets, but making analysis of data as widely accessible as possible.  
```

Comment:
Should this be the first point? Reads weird to me to have this as the first point :)

Basically, I think that the mission statement needs like 2 introductory sentences IMO.

Or move the values below the abstract.

Reply:
On positioning for values points – Not sure, had just drafted some points and wanted them in here for comment

I think we're known for efficiency, and to me it's a inclusivity issue (e.g. if you have unnecessarily high hardware requirements you are putting limitations on who can do science). I think it could use rephrasing/ ordering, but it has been a core value.